### PR TITLE
MGMT-4092 Update AI to set bootStrapInPlace option in the install con…

### PR DESCRIPTION
**Pull request by tsorya**
…fig for SNO

### CODE REVIEW

File: internal/installcfg/installcfg_test.go [hunk 1]
```diff
@@ -269,6 +269,25 @@ var _ = Describe("installcfg", func() {
...
+		cluster.HighAvailabilityMode = &mode
+		cluster.Hosts[0].Bootstrap = true
+		cluster.Hosts[0].InstallationDiskPath = "/dev/test"
+
+		data, err := GetInstallConfig(logrus.New(), &cluster, false, "")
+		Expect(err).ShouldNot(HaveOccurred())
+		err = yaml.Unmarshal(data, &result)
+		Expect(err).ShouldNot(HaveOccurred())
+		Expect(result.Platform.Baremetal).Should(BeNil())
+		var none = platformNone{}
```
Comment (osherdp) [inline internal/installcfg/installcfg_test.go:286]: inline

File: internal/common/common.go [hunk 2]
```diff
@@ -138,30 +139,47 @@ func AllStrings(vs []string, f func(string) bool) bool {
...
+func GetBootstrapHost(cluster *Cluster) *models.Host {
+	for _, host := range cluster.Hosts {
+		if host.Bootstrap {
+			return host
+		}
+	}
+	return nil
+}
+
+// IsSingleNodeCluster if this cluster is sno or not
```
Comment (osherdp) [inline internal/common/common.go:182]:
```suggestion
// IsSingleNodeCluster if this cluster is single-node or not
```

**Review by osherdp — CHANGES_REQUESTED** [review]

**Review by eranco74 — COMMENTED** [review]
/lgtm

**Pull request merged**